gm4_heart_canisters/beet.yaml

```diff
@@ -19,17 +19,17 @@ meta:
         lib_custom_crafters: 3.4.0
       schedule_loops: [main]
     model_data:
-      - item: poisonous_potato
+      - item: player_head
```

Comment:
Why were these models changed back to player heads?

Reply:
The way the RP works it uses CMD on the vanilla item model, so since the heart canisters use the player_head item_model it needs to alter that instead of the poisonous potato for the RP to work

Reply:
Ah right of course